### PR TITLE
Add test but fail to fix

### PR DESCRIPTION
**Pull request by gannetson**
If applicable

- [ ] Added translatable strings to `server-deployment`
- [ ] Added translations to `sever-deployment`

### CODE REVIEW

File: bluebottle/bluebottle_dashboard/urls.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import django
```
Comment (eodolphi) [inline bluebottle/bluebottle_dashboard/urls.py:1]: Why is this here? The monkey patching should be enough right? And why do we mark these urls as `csrf_exempt`?

Comment (gannetson) [reply bluebottle/bluebottle_dashboard/urls.py:1]: Remove this file. We are not using it

Comment (gannetson) [reply bluebottle/bluebottle_dashboard/urls.py:1]: We are not using this file. Please remove it.
